components/tabs/index.css

```diff
@@ -408,6 +408,12 @@ governing permissions and limitations under the License.
 		}
 	}
 
+	&:focus:not(:focus-visible) {
+		&::before {
+			border-color: transparent;
+		}
+	}  
+
 	&:focus,
```

Comment:
I _think_ you can achieve the same by simply removing this line, instead?

Reply:
I tried that but it didn't seem to work as I expected. I may have done something incorrect though, so I'll try again!

Reply:
chrome leaves an ```outline``` property maybe you can try with that.

Reply:
Yeah, removing both instances of `:focus` within the CSS should be the way to go here. `focus-visible` will not trigger on click of the divs that have `tabindex="0"`.